code/game/objects/effects/landmarks.dm

```diff
@@ -408,8 +408,8 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark/start/new_player)
 	layer = HIGH_LANDMARK_LAYER
 
 
-/obj/effect/landmark/event_spawn/New()
-	..()
+/obj/effect/landmark/event_spawn/Initialize(mapload)
+	. = ..()
 	GLOB.generic_event_spawns += src
```

Comment:
have you verified nothing is trying to access this list before atom subsystem inits?